tests/console/slp.pm

```diff
@@ -27,24 +27,10 @@ sub run {
     my ($self) = @_;
     select_console 'root-console';
 
-    # Workaround: slpd requires its own user
-    # %pre statement in spec file of openslp-server creates dependency to daemon group provided by *system-user-daemon*
-    # /usr/sbin/useradd -r -g daemon -d /var/lib/empty -s /sbin/nologin -c "openslp daemon" openslp 2>/dev/null || :
-    # up to this point, JeOS doesn't require *system-user-daemon*
-    if (script_run 'getent group daemon') {
-        zypper_call 'in system-user-daemon';
-        assert_script_run 'useradd -d /var/lib/empty/ -s /sbin/nologin openslp';
-        record_soft_failure 'bsc#1165050 openslp-server does not create openslp user when system-user-daemon is not installed';
-    }
+    disable_and_stop_service($self->firewall) if (script_run("which " . $self->firewall) == 0);
```

Comment:
Why does this change the order?

Reply:
I accidentally cut out that line with the soft fails and haven't returned it on the same spot. Good that you are asking. Fixed!